docs/source/toc.rst

```diff
@@ -17,6 +17,12 @@
 
    gamestream/gamestream
 
+.. toctree::
```

Comment:
@ReenigneArcher Let me know where this subsection should belong :smile:

Reply:
I'm thinking after `Usage` and maybe it should include `App Examples`.

Reply:
Like this?  
![2023-08-22-211124_301x923_scrot](https://github.com/LizardByte/Sunshine/assets/16567905/3258823a-c13f-41cf-8716-9153c7d88045)

Reply:
Looks fine to me.